addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -152,15 +152,23 @@
                     <div t-att-class="'o_wslides_lesson_aside_list_link d-flex p-1 %s%s' % (('bg-100 active' if aside_slide == slide else ''), 'text-muted' if not can_access else '')"
                         t-att-data-id="slide.id"
                         t-att-data-completed="slide_completed">
+                        <t t-set="slide_icon" t-if="aside_slide.slide_type == 'pdf'" t-value="'fa-file-pdf-o'"/>
```

Comment:
A bit sad it is basically a duplicate of "slide_icon" content, but I don't see much options to factorize it in templates, as those are not really method calls ... maybe we could have a slide_icon computed field when reaching master, what do you think ?

Reply:
Hi @tde-banana-odoo :slightly_smiling_face: Thanks for the review!
I added a new branch and PR on the pad for the master version :slightly_smiling_face: 
Also, we had an interrogation with jbn : https://github.com/odoo/odoo/pull/121179#discussion_r1200522493
What do you think about it ? :slightly_smiling_face: 
Thank you!

Reply:
I propose to do the forward-port of this one until master, then make the master improvement on top of it (easier for tracking history) :) .

Reply:
I created a task for this here :slightly_smiling_face:  : https://www.odoo.com/web#menu_id=4720&cids=1&action=333&active_id=4872&model=project.task&view_type=form&id=3378180